systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -416,54 +413,59 @@ void testUOListeningOnlyUsersInSameCluster(ExtensionContext extensionContext) {
 
-        LOGGER.info("Verifying KafkaUser: {}/{} has label: {}={} corresponding to Kafka Cluster it belongs to", namespace, testStorage.getUsername(), Labels.STRIMZI_CLUSTER_LABEL, userListeningClusterName);
-        String kafkaUserResource = cmdKubeClient(namespace).getResourceAsYaml("kafkauser", testStorage.getUsername());
+        LOGGER.info("Verifying KafkaUser: {}/{} has label: {}={} corresponding to Kafka Cluster it belongs to", clusterOperator.getDeploymentNamespace(), testStorage.getUsername(), Labels.STRIMZI_CLUSTER_LABEL, userListeningClusterName);
+        String kafkaUserResource = cmdKubeClient(clusterOperator.getDeploymentNamespace()).getResourceAsYaml("kafkauser", testStorage.getUsername());
         assertThat(kafkaUserResource, containsString(Labels.STRIMZI_CLUSTER_LABEL + ": " + userListeningClusterName));
 
-        SecretUtils.waitForSpecificLabelKeyValue(testStorage.getUsername(), namespace, Labels.STRIMZI_CLUSTER_LABEL, userListeningClusterName);
+        SecretUtils.waitForSpecificLabelKeyValue(testStorage.getUsername(), clusterOperator.getDeploymentNamespace(), Labels.STRIMZI_CLUSTER_LABEL, userListeningClusterName);
 
-        KafkaUserUtils.waitForKafkaUserMappingIntoKafkaResource(namespace, testStorage.getUsername(), userListeningClusterName, scraperPodName);
+        KafkaUserUtils.waitForKafkaUserMappingIntoKafkaResource(clusterOperator.getDeploymentNamespace(), testStorage.getUsername(), userListeningClusterName, scraperPodName);
 
-        LOGGER.info("Verifying KafkaUser: {}/{} is not present in Kafka ecosystem in Kafka/{}", namespace, testStorage.getUsername(), userIgnoringClusterName);
-        String getUserResult = KafkaCmdClient.describeUserUsingPodCli(namespace, scraperPodName, KafkaResources.plainBootstrapAddress(userIgnoringClusterName), "CN=" + testStorage.getUsername());
+        LOGGER.info("Verifying KafkaUser: {}/{} is not present in Kafka ecosystem in Kafka/{}", clusterOperator.getDeploymentNamespace(), testStorage.getUsername(), userIgnoringClusterName);
+        String getUserResult = KafkaCmdClient.describeUserUsingPodCli(clusterOperator.getDeploymentNamespace(), scraperPodName, KafkaResources.plainBootstrapAddress(userIgnoringClusterName), "CN=" + testStorage.getUsername());
         assertThat("KafkaUser CR is not mapped as Kafka resource user", getUserResult, not(containsString(testStorage.getUsername())));
     }
 
     @BeforeAll
     void setup(ExtensionContext extensionContext) {
+        this.clusterOperator = this.clusterOperator
+            .defaultInstallation(extensionContext)
+            .createInstallation()
+            .runInstallation();
+
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(userClusterName, 1, 1)
             .editMetadata()
-                .withNamespace(namespace)
+                .withNamespace(clusterOperator.getDeploymentNamespace())
```

Comment:
out of curiosity, do we still need to specify the namespace, when it is now always same?

Reply:
I think not, but maybe there are edge cases, which could switch the namespace context. (e.g., MultipleNamespaceST...). Anyway, I can try it and if it works, we can delete such occurrences in the code... :)